src/dynamics/tracers.jl

```diff
@@ -0,0 +1,52 @@
+abstract type AbstractTracer <: AbstractModelComponent end
+
+export Tracer
+@kwdef mutable struct Tracer <: AbstractTracer
+    name::Symbol
+    active::Bool = true
+end
+
+Tracer(name::Symbol; kwargs...) = Tracer(; name, kwargs...)
+
+const TRACER_DICT = Dict{Symbol, AbstractTracer}
+
+function add!(dict::TRACER_DICT, tracers::Tracer...)
+    for tracer in tracers
+        dict[tracer.name] = tracer
+    end
+    return dict
+end
```

Comment:
I am a bit confused, where is this method of `add!` with the `TRACER_DICT` used? I find it.